tools/internal_ci/helper_scripts/prepare_build_linux_rc

```diff
@@ -41,4 +41,29 @@ export DOCKERHUB_ORGANIZATION=grpctesting
 
 git submodule update --init
 
+if [ "${PREPARE_BUILD_INSTALL_DEPS_PYTHON}" == "true" ]
+then
+  # Prepares the build dependencies
+  sudo apt update
+  sudo apt install -y zlib1g-dev libssl-dev jq build-essential libffi-dev wget
+
+  # Downloads and compiles Python 3.9.12
+  pushd /tmp
+  wget -q https://www.python.org/ftp/python/3.9.12/Python-3.9.12.tgz
```

Comment:
Optional: Copy this to one of our buckets in case this server goes down or the URL changes?

Reply:
Good idea! Let me add that.